### ISSUE TITLE
Drawer: lack of accessibility

### ISSUE THREAD

**Issue by MortenHofft** [gbif/gbif-web#678]
I suggest we replace the plain css implementation with radix or similar. There are many accessibility concerns. This would probably also solve

https://github.com/gbif/gbif-web/issues/667
https://github.com/gbif/gbif-web/issues/666
https://github.com/gbif/gbif-web/issues/664
